demos/local-only-todolist/lib/models/schema.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:powersync/powersync.dart';
+import 'package:powersync_flutter_local_only_demo/models/sync_mode.dart';
+
+/// The schema contains two copies of each table - a local-only one, and
+/// a online/synced one. Depending on the 'online' flag, one of those gets
+/// the main 'lists' / 'todos' view name.
+///
+/// For local only, the views become:
+///   online_todos
```

Comment:
I think this is switched with the online version?

Reply:
Never mind, had to do some mental processing for this.

Reply:
This caught me too, we should perhaps expand the comment for a bit of hand holding?